src/json/balances/balance_compound_ether.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tokenAddress": "0xd6801a1DfFCd0a410336Ef88DeF4320D6DF1883e",
+  "token": {
+    "name": "Compound Ether 📈",
+    "symbol": "cETH",
+    "decimals": 8,
+    "logoUri": "https://gnosis-safe-token-logos.s3.amazonaws.com/0xd6801a1DfFCd0a410336Ef88DeF4320D6DF1883e.png"
+  },
+  "balance": "5002",
```

Comment:
while they are correct, I would remove the `balanceUsd` and `usdConversion` in favor of the `fiatX` methods and also use them in the backend model